judge/pdf_problems.py

```diff
@@ -32,6 +42,12 @@ def load(self, file, source):
             target.write(source.read())
 
     def make(self, debug=False):
+        self._make(debug)
+
+        if self.title and HAS_EXIFTOOL:
+            subprocess.call([EXIFTOOL, '-Title=%s' % (self.title,), self.pdffile])
```

Comment:
The lack of logging here has me concerned.

Reply:
I was thinking that it's unimportant since nothing really happens if `exiftool` dies, but yea, probably should log it...

Reply:
Added logging for `exiftool` errors.